tr55/model.py

```diff
@@ -241,15 +241,16 @@ def create_modified_census(census):
                 else:
                     raise Exception('Unknown modification type.')
 
-                parent['distribution'] = {
-                    cell: {"cell_count": n - m},
-                    changed_cell: {"cell_count": m}
-                }
+                if 'distribution' not in parent:
+                    parent['distribution'] = {cell: {'cell_count': n}}
+
+                parent['distribution'][cell]['cell_count'] -= m
+                parent['distribution'][changed_cell] = {'cell_count': m}
 
     return mod
 
 
-def simulate_water_quality(tree, cell_res, fn, current_cell=None, precolumbian=False):
+def simulate_water_quality(tree, cell_res, fn, current_cell=None, precolumbian=False):  # noqa
```

Comment:
If these `noqa` comments are due to line length complaints, you can always break the arguments to another line.  I'd also be ok with increasing the line length lint settings to > 80.